### DISCUSSION TITLE
Path does not exist in JsonFilter

### DISCUSSION THREAD

**Discussion by LewisMorgans** [prisma/prisma#10592] [Q&A] [answered]
Hey guys. 

Not sure what's up. But path keeps erroring for me. 

```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = [ "mongoDb", "filterJson"]
}

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}

model User {
  id       String @id @default(dbgenerated()) @map("_id") @db.ObjectId
  email    String
  options  Json
  password String
  role     String
}
```

```
Invalid `.findMany()` invocation 

for (const [key, value] of Object.entries(filters)) {
     return await prisma.user
→     .findMany({
           where: {
             options: {
               path: key,
               ~~~~
               equals: value
             }
           }
         })

Unknown arg `path` in where.options.path for type JsonFilter. Did you mean `not`? Available args:
type JsonFilter {
  equals?: Json
  not?: Json
}

```

Have ran npx prisma generate several times.

Any help?

**Comment by matthewmueller** [comment, marked as answer]
Hey @LewisMorgans , this isn't available yet. Do you mind creating a [feature request](https://github.com/prisma/prisma/issues/new/choose)? Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
